packages/keycloak-user-management/src/mls.tsx

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable @typescript-eslint/camelcase */
```

Comment:
If this file is going to be included in all packages could we document this so that an engineer is aware that this file needs to be created when they build a new package